### DISCUSSION TITLE
Application Goals Discussion

### DISCUSSION THREAD

**Discussion by wienans** [wienans/quadcoach#14] [Announcements]
Main focus is to be a tool for Coaches.

- coaches can use the tool to look up exercises and build a session
- Create Tactics with a tactics board an export them as animation to share with the team.

**Comment by wienans** [comment]
Based on the feedback a online application hosted for all at one place ist preferable, most people wouldn’t be able to use the application.

**Reply by JoramStiel** [reply]
Not sure how many people would use it then, but we might need some control that we stay in our limits.
For starting we could compare [Vercel ](https://vercel.com/) and [render](https://render.com/)

**Comment by wienans** [comment]
Based on that Account Management needs to be implemented #15 

Then we would need to think about public an private databases for tactics and exercises. 
should everything be public? Most said they would share most to all exercises public. For tactics that might change. 
sharing one huge database would be the easiest to store and have the lowest redundancy in storage. 
But then we need at least a favourite system which is stored for every user.

**Reply by JoramStiel** [reply]
Yeah, I think we would need a role system. Maybe admins (we and maybe who ever) and maybe deleting a exercise could only do creator and we. Unfortunatelly I think mainly we need a role system or something so that if some exercises which are not exercise but using it as platform to post whatever. So we would need a system to let users vote for ban of this exercise.

**Comment by JoramStiel** [comment]
One idea I had today was that if multiple people would host the application you could store exercise on several hosts and then implement a search for the hosts and you could open their exercise, then we wouldn't have the data on only one database :D

**Reply by wienans** [reply]
The thing is people will likely not host it themselves it was a strong minority who said they might be able to do it

**Comment by JoramStiel** [comment]
What I also would like to have is to create trainings and posibility to plan them and see a history of trainings. This would definitly need a role and user system. 
So you could say today I have x Chase, y Beater, z Keeper, a Seeker and then you could click exercise together on a time schedule. In future  we might use something to generate it automatically
